packages/tool.release/src/simulateMinorBump.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2025 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import fs from "node:fs";
+import path from "node:path";
+import * as semver from "semver";
+
+export function simulateMinorBump(): void {
+  const cwd = process.cwd();
+
+  const packagesDir = path.join(cwd, "./packages");
+  const changesetsDir = path.join(cwd, "./.changeset");
```

Comment:
fyi most tools are fine with just `./packages` if you want but I do like being explicit cause it makes the code easier to refactor and use for other things.